src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0001Tests.cs

```diff
@@ -22,8 +22,8 @@ public class Program { }
             var diagnostics = await _runner.GetDiagnosticsAsync(testSource.Source);
             var diagnostic = Assert.Single(diagnostics);
             Assert.Equal("AZC0001", diagnostic.Id);
-            Assert.Equal("Namespace 'RandomNamespace' shouldn't contain public types. Use one of the following pre-approved namespace groups:" +
-                         " Azure.Diagnostics, Azure.Cognitive, Azure.Iot, Azure.Networking, Azure.Runtime, Azure.Security, Azure.Storage, Azure.ApplicationModel", diagnostic.GetMessage());
```

Comment:
We should leave Azure.ApplicationModel. It was recently decided to leave that namespace for Preview 2 and then later do the renaming

Reply:
Done